server/src/main/resources/config/nodeSauceConfig.json

```diff
@@ -2,16 +2,8 @@
     "class": "org.openqa.grid.common.RegistrationRequest",
     "capabilities": [{
         "platform": "ANY",
-        "browserName": "firefox",
-        "maxInstances": 20
-    }, {
-        "platform": "ANY",
-        "browserName": "chrome",
-        "maxInstances": 20
-    }, {
-        "platform": "ANY",
-        "browserName": "internet explorer",
-        "maxInstances": 20
+        "browserName": "saucelabs",
```

Comment:
Note: I'll make a change here that sets the number of saucelabs slots to the max concurrent VM's the suace labs account has.